src/lib/types/domain/song.ts

```diff
@@ -1,14 +1,19 @@
 import type { Timestamp, DocumentReference } from 'firebase/firestore'
 
-export type song = {
+export type Song = {
 	id: string
 	name: string
 	artist: string
-	length: string
 	link: string
+}
+
+export type SongSuggestion = Omit<SuggestedSong, 'id' | 'liked'>
+
+export type SuggestedSong = Song & {
+	length: string
```

Comment:
length moet deel zijn van ieder nummer

Reply:
en genre ook

Reply:
Waarom? Werkt prima vgm zo?

Reply:
Zoals ik het nu begrijp, is song gewoon een nummer, en suggestedsong een nummer dat ingestuurd is. Een normale song heeft sws een length en genre, dus de commissie moet dat ook gewoon invullen

Reply:
Nee, het genre en lengte is alleen belangrijk voor het behandelen van suggesties door de commissie op setlistvergaderingen. Daarom horen die bij `SuggestedSong` erbij. Als een nummer op de setlist staat, is die info (op de site) niet meer relevant, dus volstaat een `Song`.

Reply:
Tis ook belangrijk voor setlist inzichten